### DISCUSSION TITLE
How do I delete an old server instance's cached account information?

### DISCUSSION THREAD

**Discussion by jamescridland** [mastodon/mastodon#22714] [Q&A]
So, if you do a search for `@james@crid.land` on some servers you'll get to, for example...

https://mastodon.social/@james@socl.crid.land

This information looks like it lives at `socl.crid.land`. It doesn't. I tried to get to it work well but failed dismally, and in the end the server died in a blaze of overloaded AWS unpleasantness.

I've set the webfinger correctly for `@james@crid.land` and also for `@james@socl.crid.land` so that it points to my real home on `@james@bne.social` but **will these cached ghost accounts live on forever throughout the fediverse, or is there anything I can do to indicate to the fediverse that these are old accounts and shouldn't exist any more?**

**Comment by PeacheyMcKeitch** [comment]
Also wondering this! On activity pub via micro.blog and I changed my mastodon compatible URL a few times. No mastodon.social for example has three different entries for me in search.

**Comment by nightpool** [comment]
You'll need to set the `movedTo` URL on the profile that you want to redirect, and then set the `alsoKnownAt` URL on the destination account (to authorize the link). Then you can send an `Update` or `Move` activity to all of the servers that you want to be updated and they'll show the account as having been "moved to" the new account.

**Reply by jamescridland** [reply]
This would be the answer to "I want to redirect a server".

I'd like to remove cached information for a server that is no longer running. How do I do that?

**Comment by cuprevenge** [comment]
https://github.com/mastodon/mastodon/issues/10305
https://docs.joinmastodon.org/admin/tootctl/#accounts-cull

I hope the above two links will help you.

**Reply by jamescridland** [reply]
This is all great new functionality - from a server administrator's point of view.

Sadly it doesn't help me get rid of the detritus when my server froze and died - it still seems searchable all over the fediverse (and, more to the point, still seems active from the search).

**Comment by jamescridland** [comment]
Is one way to achieve this to produce a NEW server at the old address of socl.crid.land - and then to close it properly and correctly? It's still there on a number of servers, it appears.

OR, is it possible just to forward that domain to the new server (thankfully, the account name is the same), and would that be seen by an ActivityPub server as a redirect?

**Reply by PeacheyMcKeitch** [reply]
Yeah I’m trying that out just now. I made a new single user server matching the old mastodon username exactly. And it kinda works in a scattershot kinda way.

Set up the server, changed the display name to “my name (old account)” and a few servers picked it up. Also put in my new username in the server description so if people follow links to it they will see that as well.

I found that I had to go follow a random person on each effected server sometimes for it to be picked it up. It’s only been up for three days though so maybe it’ll federate out with time.

I’ve set the redirect-server account to migrate to my new one now. Hopefully can delete it outright in a week or two. Though bear in mind that once you set your account to migrate you can’t really use the server for anything useful without setting up another account.

**Comment by jamescridland** [comment]
I'd really like to see if this is possible to fix these days. Can anyone help?
